README.md

```diff
@@ -0,0 +1,64 @@
+# Portfolio-mobile-version
+Microverse project
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [ ] it is good that you used[ the Microverse template](https://github.com/microverseinc/readme-template) for your readme file however please make sure to fill it in with the information related to you and your project. keep in mind that all sections in template are mandatory apart from the "live demo", "Run tests" and "Deployment" (unless these are a requirement in a specific project)